corehq/apps/es/es_query.py

```diff
@@ -205,10 +205,17 @@ def __getitem__(self, sliced_or_int):
             size = sliced_or_int.stop - start
         return self.start(start).size(size).run().hits
 
-    def run(self):
+    def run(self, no_hits_with_aggs=True):
```

Comment:
If anyone ever used this, I'd suggest a different name, like `include_hits=None` and then below `if include_hits is None: include_hits = query.uses_aggregations()` and `if not include_hits: query.size(0)`. That way the way you call it is `.run(include_hits=True)` (if using aggs, otherwise no need to call it).

On the other hand, it looks like it's not used anywhere. I know you and @esoergel talked about it, but if that's the case I'm tempted to YAGNI it.